roles/openshift_docker_registry/tasks/main.yml

```diff
@@ -0,0 +1,11 @@
+---
+
+# TODO: Centralized openshift_config openshift_facts??
+# TODO: anchor this on a master
+- name: Query for OpenShift Docker Registry
+  command: openshift admin registry --credentials={{ openshift_config }}
+  register: registry_results
```

Comment:
The command module always says something changed, even when it didn't.

Since we know that this command will never change anything (it's just a query), adding this line will make it say the system wasn't changed:
```
changed_when: False
```